### PR TITLE
Ampstart sample fixes

### PR DESCRIPTION
**Pull request by sebastianbenz**
I went through all samples to check their formatting after the ampstart based re-design. Things I fixed:

* broken layout / samples
* removed header from NewsArticle / Live Blog sample as it didn't provide any value
* fixed form formatting
* improve layouts 
* add info background color
* clean up of unused css rules

Every commit targets a single sample / fix. Best thing to do for review is go through each commit one by one.

### CODE REVIEW

File: src/30_Advanced/How_to_create_interactive_AMP_pages%3F.html [hunk 1]
```diff
@@ -34,16 +34,16 @@
   <!--
     The key is to split your webpage into content that is defined with AMP...
   -->
-  <div>
-    <h4>Who's going to win the Oscars?</h4>
+  <section>
+    <h2>Who's going to win the Oscars?</h2>
```
Comment (sebastianbenz) [inline src/30_Advanced/How_to_create_interactive_AMP_pages%3F.html]: You mean in the preview?

Comment (sebastianbenz) [inline src/30_Advanced/How_to_create_interactive_AMP_pages%3F.html]: done

File: src/30_Advanced/Star_Rating.html [hunk 2]
```diff
@@ -47,7 +47,6 @@
       -webkit-tap-highlight-color: rgba(0,0,0,0);
       -webkit-tap-highlight-color: transparent;
       /* border: 1px solid red;  /* debug */
-      margin-bottom: 1em;
     }
 
     /* the stars */
```
Comment (kul3r4) [inline src/30_Advanced/Star_Rating.html]: For this sample, the section "Usage" is rendered as if it was a sample but it's actually only instructions.
usage

File: src/30_Advanced/Using_the_Google_AMP_Cache.html [hunk 3]
```diff
@@ -51,7 +51,7 @@
 
 Here is AMP Cache URL for `https://ampbyexample.com/components/amp-img`:
 
-<div class="amp-iframe-container"><amp-iframe height="88" 
+<div class="amp-iframe-container"><amp-iframe height="149"
```
Comment (kul3r4) [inline src/30_Advanced/Using_the_Google_AMP_Cache.html]: Button "CONVERT" is in the wrong position
cache

Comment (sebastianbenz) [inline src/30_Advanced/Using_the_Google_AMP_Cache.html]: This is fixed, but not deployed to the api endpoint.

File: src/30_Advanced/Star_Rating.html (hunk 2 above)
Comment (sebastianbenz) [inline src/30_Advanced/Star_Rating.html]: I agree - this is weird. Problem is that these needs to be in both, code and demo view.

File: src/50_Samples_%26_Templates/Product_Listing.html [hunk 4]
```diff
@@ -99,7 +99,7 @@ <h4 class="category"><a href="/samples_templates/product_listing/preview/">More<
     </a>
   </amp-carousel>
 
-  <h5>[[.Title]]</h5>
+  <h3>[[.Title]]</h3>
```
Comment (kul3r4) [inline src/50_Samples_%26_Templates/Product_Listing.html]: This section and the one below is missing margins in demo.
listing

File: src/30_Advanced/Using_the_Google_AMP_Cache.html (hunk 3 above)
Comment (kul3r4) [inline src/30_Advanced/Using_the_Google_AMP_Cache.html]: Ok, thanks!

File: src/30_Advanced/Star_Rating.html (hunk 2 above)
Comment (kul3r4) [inline src/30_Advanced/Star_Rating.html]: ok, we could improve a little bit by:
- adding proper margins
- change "Things to try" to something like "instructions to interact with the start rating widget / to experiment the star rating widget"

Comment (sebastianbenz) [inline src/30_Advanced/Star_Rating.html]: Done. Fixed by splitting into separate doc and preview.

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

**Comment by sebastianbenz** [comment]
deploying to staging

**Pull request merged**